doc/src/manual/performance-tips.md

```diff
@@ -21,12 +21,18 @@ const DEFAULT_VAL = 0
 Uses of non-constant globals can be optimized by annotating their types at the point of use:
 
 ```julia
-global x
-y = f(x::Int + 1)
```

Comment:
This seemed like a silly example, using a type annotation to improve the performance of a single plus. Changed it to instead be something like in a loop.